### PR TITLE
Add topics for payments and interoperability usecases

### PR DESCRIPTION
**Pull request by amarantha-k**

### CODE REVIEW

File: docs/use-cases/interoperability/index.mdx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+---
+import { XRPLCardDeck, DeckCard } from "../../../_components/XRPLCardDeck";
+
+# Interoperability
+
+These use cases focus on capabilities on the XRP Ledger to interact with other chains.
+
+<XRPLCardDeck cols="1/3">
+  <DeckCard title="Sidechains (Beta)" to="sidechains.md">
+    Run an independent ledger with its own consensus algorithm and transaction types and rules.
```
Comment (DennisDawson) [inline docs/use-cases/interoperability/index.mdx]: consider "consensus algorithm, transaction types, and rules."

File: docs/use-cases/interoperability/index.mdx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  </DeckCard>
+  <DeckCard title="XRPL Hooks" to="https://xrpl-hooks.readme.io/" external>
+    Hooks add smart contract functionality to the XRP Ledger: 'layer one' custom code to influence the behaviour and flow of transactions.
```
Comment (DennisDawson) [inline docs/use-cases/interoperability/index.mdx]: What does it mean to put 'layer one' in single quotes? Should it be double quotes, or is this something in syntax?

Are we using British spelling in our docs (_behavior_)?

File: docs/use-cases/payments/index.mdx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+<XRPLCardDeck cols="1/3">
+  <DeckCard title="Payment Channels" to="payment-channels.md">
+    As an individual buying items in bulk, I want to set up a payment channel to send "asynchronous" XRP payments to a vendor, such that the payment can be divided into very small increments and settled later.
+  </DeckCard>
+  <DeckCard title="Cross-Currency Payments" to="cross-currency-payments.md">
+    As an individual buyer, I want to set up a payment channel to send cross-currency payments that exchange tokens, XRP, or both.
+  </DeckCard>
+  <DeckCard title="Escrow" to="escrow.md">
+    As an individual making a payment for a physical asset, I want to set up an escrow (conditional payments) and set aside XRP, and deliver the XRP later when certain conditions are met.
```
Comment (DennisDawson) [inline docs/use-cases/payments/index.mdx]: consider "then deliver..." so that it doesn't sound like a comma splice

**Review by DennisDawson — APPROVED** [review]
Consider my suggestions, but otherwise good to go.

File: docs/use-cases/interoperability/index.mdx (hunk 2 above)
Comment (amarantha-k) [reply docs/use-cases/interoperability/index.mdx]: Good catch. Thanks.

**Pull request merged**